docs/src/pages/[platform]/connected-components/storage/storageimage/props.ts

```diff
@@ -0,0 +1,30 @@
+export const STORAGE_IMAGE = [
+  {
+    name: 'imgKey',
```

Comment:
Can we add alt here since it's required by underlying Image?

Reply:
Makes sense